package.json

```diff
@@ -4,50 +4,63 @@
   "description": "Blazing Fast React UI Primitive",
   "contributors": [
     "Jeroen Ransijn (https://twitter.com/jeroen_ransijn)",
-    "Roland Warmerdam (https://roland.codes)"
+    "Roland Warmerdam (https://roland.codes)",
+    "Meichen Zhou (https://github.com/jfmaggie)",
+    "Netto Farah (https://twitter.com/nettofarah)"
   ],
   "keywords": [
     "react"
   ],
   "repository": "segmentio/ui-box",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
   "files": [
-    "lib"
+    "lib/src"
   ],
   "sideEffects": false,
   "scripts": {
-    "test": "xo && nyc ava",
+    "test": "yarn run tslint && xo && nyc ava",
+    "tslint": "tslint -c tslint.json 'src/**/*.ts'",
     "prepublishOnly": "rm -rf lib && yarn run build",
     "precommit": "lint-staged",
-    "dev": "start-storybook -p 9009",
-    "build": "babel --out-dir lib src --source-maps inline",
+    "dev": "start-storybook -p 9009 -c .storybook",
+    "build": "tsc --declaration",
     "build-storybook": "build-storybook -s .storybook/static -o .out",
     "release": "np --no-publish",
     "benchmark": "echo ui-box && react-benchmark tools/benchmarks/box.js && echo glamor/jsxstyle && react-benchmark tools/benchmarks/glamor-jsxstyle.js",
     "size": "size-limit",
     "coverage": "nyc report --reporter=html"
   },
   "dependencies": {
-    "@emotion/hash": "^0.6.5",
     "glamor": "^2.20.0",
     "inline-style-prefixer": "^5.0.1",
+    "murmurhash-js": "^1.0.0",
     "prop-types": "^15.6.0"
   },
   "peerDependencies": {
     "react": "^16.0.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.2.2",
     "@babel/plugin-proposal-class-properties": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
-    "@storybook/react": "^3.4.10",
+    "@storybook/react": "^4.1.4",
     "@storybook/storybook-deployer": "^2.2.0",
-    "ava": "^1.0.0-beta.8",
+    "@types/enzyme": "^3.1.15",
+    "@types/inline-style-prefixer": "^5.0.0",
+    "@types/murmurhash-js": "^1.0.3",
```

Comment:
Shouldn't the dependencies types be in the dependencies so that consumers don't need install them? https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies

Reply:
I'm seeing mixed things from the TypeScript community. On the one hand, you don't want non-TS users to have to install these, but on the other hand they are required for TS to work.

I've seen some cases of extremely liberal version requirements when in `dependencies`. I've seen them in `peerDependencies`, and I've seen them in both `devDependencies` + `optionalDependencies`.